functions.php

```diff
@@ -107,3 +107,17 @@
 if ( class_exists( 'WooCommerce' ) ) {
 	require get_parent_theme_file_path( 'inc/woocommerce.php' );
 }
+add_action( 'jarvis_header', 'jarvis_include_header' );
+/**
+ * Include the default site header for the site.
+ */
+function jarvis_include_header() {
+	include 'inc/header.php';
```

Comment:
could we change this to `get_template_part( 'parts/header.php' )`?

Using get_template_part means the files can be filtered/ overridden by child themes.

In addition template parts go in the parts directory, and code (filters and reusable functions) go in inc. I should add something to the readme.md about the folder structure.

Reply:
I have moved these to template parts.